packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -43,7 +44,8 @@ abstract class Scrollable extends StatefulWidget {
     this.onScrollStart,
     this.onScroll,
     this.onScrollEnd,
-    this.snapOffsetCallback
+    this.snapOffsetCallback,
+    this.builder
   }) : super(key: key) {
     assert(scrollDirection == Axis.vertical || scrollDirection == Axis.horizontal);
     assert(scrollAnchor == ViewportAnchor.start || scrollAnchor == ViewportAnchor.end);
```

Comment:
We should add a comment to the four places that duplicate all these docs listing the four classes that have those docs and telling future maintainers that they must ensure the four places are kept in sync.